bitnami/rabbitmq/README.md

```diff
@@ -263,35 +263,31 @@ extraConfiguration: |
   log.console.formatter = json
 ```
 
-## How to Avoid Deadlocked Deployments After a Cluster-Wide Restart
+### How to Avoid Deadlocked Deployments After a Cluster-Wide Restart
 
-RabbitMQ nodes assume their peers come back online within five minutes (by default). When the `OrderedReady` pod management policy is used
-with a readiness probe that implicitly requires a fully booted node, the deployment can deadlock:
+RabbitMQ nodes assume their peers come back online within five minutes (by default). When the `OrderedReady` pod management policy is used with a readiness probe that implicitly requires a fully booted node, the deployment can deadlock:
 
 - Kubernetes will expect the first node to pass a readiness probe
 - The readiness probe may require a fully booted node
 - The node will fully boot after it detects that its peers have come online
 - Kubernetes will not start any more pods until the first one boots
 
-Using [RabbitMQ Cluster Operator](https://www.rabbitmq.com/kubernetes/operator/operator-overview) is the easiest solution.
+Using [Bitnami RabbitMQ Cluster Operator](https://hub.docker.com/r/bitnamicharts/rabbitmq-cluster-operator) is the easiest solution.
```

Comment:
If this is [the RabbitMQ core team's Operator](https://www.rabbitmq.com/kubernetes/operator/operator-overview) then it's OK to recommend but I don't understand what doe Bitnami have to do with it. It was not developed by Bitnami and our team never refers to it as the "Bitnami Operator" for that reason.

If that is a different Operator, then I must say that I have never signed up for recommending a different one and the sentence must be taken out completely.

Reply:
Someone from the core team has confirmed it to me that Bitnami engineers did not contribute to https://www.rabbitmq.com/kubernetes/operator/operator-overview.

@carrodher it is NOT OK to call a project not developed by Bitnami "a Bitnami Operator".